ns-app/scratch/webrtc_test/trace_player.cc

```diff
@@ -40,6 +42,11 @@ void TracePlayer::LoadTrace() {
         TraceItem ti;
         ti.capacity_ = lexical_cast<decltype(ti.capacity_)>(trace["capacity"]);
         ti.duration_ms_ = lexical_cast<decltype(ti.duration_ms_)>(trace["duration"]);
+        if (trace.find("rtt") != trace.end()) {
+            ti.rtt_ms_ = lexical_cast<decltype(ti.rtt_ms_)>(trace["rtt"]);
+        } else {
+            ti.rtt_ms_ = 0;
```

Comment:
Don't set 0 as the default value, try boost::optional.

Reply:
fixed in commit 03f42d6